core/backend/src/test/HubMock.ts

```diff
@@ -162,14 +162,14 @@ export class HubMock {
 
   public static async getMyBriefcaseIds(arg: IModelIdArg): Promise<number[]> {
     const user = arg.user ?? await AuthorizedBackendRequestContext.create();
-    return this.findLocalHub(arg.iModelId).getBriefcaseIds(user.accessToken.getUserInfo()!.id);
+    return this.findLocalHub(arg.iModelId).getBriefcaseIds(user.accessToken ?? "");
   }
 
   public static async acquireNewBriefcaseId(arg: IModelIdArg): Promise<number> {
     const user = arg.user ?? await AuthorizedBackendRequestContext.create();

```

Comment:
With the removal of `UserInfo` from `AccessToken`, `ClientRequestContext` no longer represents a "user", making `UserArg.user` a bit of a misnomer.

This property was previously titled `requestContext` [when it was a part of `IModelNameArg`, `IModelIdArg`, a nd `BriefcaseDbArg`](https://github.com/imodeljs/imodeljs/pull/2172/files#diff-c7b5e0bba7c98837b619a88424a5baa9503b96dd56a0ad78a331c10964fb6047L56-L80), does it make sense to restore this convention as a part of this PR?

Reply:
Yea, I have a PR that is going to change this up a bit sine we'll be splitting the AccessToken from the req context.